lib/svg-sprite/shape.js

```diff
@@ -314,13 +315,29 @@ SVGShape.prototype.setSVG = function(svg) {
  */
 SVGShape.prototype._initSVG = function() {
     // Basic check for basic SVG file structure
-    const svgStart = this.svg.current.match(/<svg(?:\s+[a-z\d-:]+=("|').*?\1)*\s*(?:(\/)|(>[^]*<\/svg))>/i);
+    const validSVGRegExp = /<svg(?:\s+[a-z\d-:]+=(["']).*?\1)*\s*(?:(\/)|(>[^]*<\/svg))>/i;
```

Comment:
So, the only change in the regex is `"|'` to `["']`?

Reply:
I thought we would need to make the regex a multiline one with the `m` flag.

Reply:
@XhmikosR yes, the only change is that.
Just changing the flag of this regexp did not help, this tests are failing